api/pkg/transformer/feast/transformer.go

```diff
@@ -82,19 +74,17 @@ func (f *FeastTransformer) TransformHandler(w http.ResponseWriter, r *http.Reque
 			Entities: entities,
 			Features: features,
 		}
+		t.logger.Debug("feast_request", zap.Any("feast_request", feastRequest))
 
-		feastResponse, err := f.feastClient.GetOnlineFeatures(ctx, &feastRequest)
+		feastResponse, err := t.feastClient.GetOnlineFeatures(ctx, &feastRequest)
```

Comment:
This will make the call to feast sequential. We should use scatter and gather pattern for the external call.